src/HomePage.js

```diff
@@ -1,12 +1,47 @@
 import React from 'react';
 
+import Card from 'react-bootstrap/Card';
+import DropdownButton from 'react-bootstrap/DropdownButton';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import Container from 'react-bootstrap/Container';
+
```

Comment:
In a newer version of the Training 2 instruction, point 2 of the document addresses the format that we will be using for these import, please take a look there